src/main/scala/cromwell/binding/values/WdlValue.scala

```diff
@@ -44,4 +46,9 @@ trait WdlValue {
   def collectAsSeq[T <: WdlValue](filterFn: PartialFunction[WdlValue, T]): Seq[T] = {
     if (filterFn.isDefinedAt(this)) Seq(filterFn(this)) else Nil
   }
+
+  /**
+   * @return MD5 Hash.
+   */
+  def getHash(implicit hasher: FileHasher): Hash = valueString.md5Sum
```

Comment:
Yeah probably at this point no one really cares that it's MD5, but at the time it felt like a detail worth mentioning. Maybe in the future if we need something a bit more generic with different possible hash implementations.